changelog/undistributed/20220121122230

```diff
@@ -0,0 +1,8 @@
+--------------------------------------------------------------------------------
+                            Fix
+--------------------------------------------------------------------------------
+* IOSXE
+    * modified show_radius>:
```

Comment:
Please use name of class

```suggestion
    * Modified ShowRadiusStatistics:
```

Reply:
Please also include the change to ShowRadiusServerGroupAll too

Reply:
These changes have been done and the tests have been fixed so that the issue I had shouldn't occur for anyone else.